src/scripts/main.js

```diff
@@ -1,7 +1,36 @@
 'use strict';
 
+const body = document.querySelector('body');
+
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const box = document.createElement('div');
+
+  box.className = `notification ${
+    (type === 'warning' && 'warning') ||
+    (type === 'success' && 'success') ||
+    (type === 'error' && 'error')
+  }`;
```

Comment:
```suggestion
  box.className = `notification ${type}`;
```
What is the reason for these conditions if the type is the same as the class?
In case you need others it's better to create a map.
```javascript
const NOTIFICATION_CLASSES_MAP = {
  warning: 'some warning class',
  success: 'some success class',
  error: 'some error class',
};

box.classname = `notification ${NOTIFICATION_CLASSES_MAP[type]}`;
```